server/mailet/integration-testing/src/test/java/org/apache/james/mailets/TemporaryJamesServer.java

```diff
@@ -84,4 +87,10 @@ public void shutdown() {
         return jamesServer.getProbe(probe);
     }
 
+    private class WebAdminConfigurationModule extends AbstractModule {
+        @Override
+        protected void configure() {
+            bind(WebAdminConfiguration.class).toProvider(WebAdminConfiguration::testingConfiguration);
```

Comment:
sorry to bother you, but testConfiguration has nothing to do in a production class

Reply:
I've done on separated commit, please take a look. Thanks